### ISSUE TITLE
Threat Response IROH API - There was a client error in the AMP module: guid must be in the correct guid format.

### ISSUE THREAD

**Issue by SMSecOps** [CiscoSecurity/sxo-05-security-workflows#255]
Hi @mavander,

I imported the “0011-Talos-SolarWindsInvestigation” workflow to SecureX and configured it as instructed from [this post](https://ciscosecurity.github.io/sxo-05-security-workflows/workflows/talos/0011-solarwinds-investigation).  After running the workflow, it failed at the “Threat Response - Trigger Response Action” step. I received the below error message. I used the correct AMP module instance ID extracted from the “List Response Actions” sub-workflow? Can you please help resolve this issue?

Error Message:
`{
  "errors": [
    {
      "module_instance_id": "ae47d29e-92c7-4163-8f22-bdb358******",
      "module_type_id": "03d8591b-8374-5666-8fcc-eaea91******",
      "code": "client-error",
      "message": "There was a client error in the AMP module: guid must be in the correct guid format [XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX]: null [400]",
      "type": "error",
      "module": "Secure Endpoint - **************************"
    }
  ]
}`

List of Response Actions:

`[{"module":"Secure Endpoint - *****************************","module_instance_id":"ae47d29e-92c7-4163-8f22-bdb358******","module_type_id":"03d8591b-8374-5666-8fcc-eaea91******","id":"amp-add-sha256-scd","title":"Add SHA256 to custom detections My Custom Detection","description":"Add SHA256 to simple custom detections list My Custom Detection to detect and quarantine","url":"/respond/trigger/ae47d29e-92c7-4163-8f22-bdb358******/amp-add-sha256-scd?list=b21c0ef2-95ed-4c7a-86f5-bfde6b******&observable_type=sha256&observable_value=ee4ec2a1d909afe0feaffadfef2d9785e447a48217c573c9fd9543743d4fd0da",...}]`

**Comment by SMSecOps** [comment]
Can someone help me with this?
@msprunck @ereteog @jyoverma @Tempos

**Comment by SMSecOps** [comment]
Hi Matt,

Yes, you are right! The atomic workflow is outdated, so I ran it manually to extract the module ID based on a different module name "Secure Endpoint - *****". Then, I inputted it into the Threat Response - Trigger Response Action atomic workflow and received the incorrect GUID format error message. I opened a TAC case SR 696042951 two weeks ago but they haven't been able to help me.

- Delvain
